lib/hanami/environment.rb

```diff
@@ -52,7 +52,12 @@ class Environment
     #
     # @since 0.2.0
     # @api private
-    DEFAULT_DOTENV_ENV = '.env.%s'.freeze
+    DEFAULT_DOTENV_ENV_FILES = [
```

Comment:
WDYT about `DEFAULT_DOTENV_ENV_PATTERNS`?

Reply:
I like that better. I'll make the change.